lib/Lower/Bridge.cpp

```diff
@@ -28,6 +28,16 @@
 #include "mlir/Target/LLVMIR.h"
 #include "llvm/Support/CommandLine.h"
 
+#undef TODO
+#define TODO()                                                                 \
+  {                                                                            \
+    if (disableToDoAssertions)                                                 \
+      mlir::emitError(toLocation(), __FILE__)                                  \
+          << ":" << __LINE__ << " not implemented";                            \
+    else                                                                       \
+      assert(false && "not yet implemented");                                  \
```

Comment:
We should start using `llvm_unreachable("message")` instead of assert(false) in cases like this unless we really only want to raise failures when its built with CMAKE_BUILD_TYPE=Debug.

Reply:
I believe you intended this one to be temporary by design.  (There was an odd duplicate somewhere that I removed.)  I don't see any other "assert(false ..." cases in these files.  I do see a few in other files.